docs/Tutorial/getting-started.md

```diff
@@ -0,0 +1,128 @@
+# Quick guide
+
+## What you'll do
+
+- Deploy the nginx-ingress-integrator charm
+- Relate it to another charm
+- Inspect the ingress it creates.
+
+We'll then also look at changing that ingress via a juju configuration update.
+
+## Requirements
+
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed. We’ll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
```

Comment:
s/We’ll also want to make/Make
s/which we can do by/by

As a general rule, keep the grammar as simple as possible ;)